src/otx/algo/classification/vit.py

```diff
@@ -494,10 +494,11 @@ def _build_model(self, head_config: dict) -> nn.Module:
         return ImageClassifier(
             backbone=vit_backbone,
             neck=None,
-            head=HierarchicalLinearClsHead(
+            head=HierarchicalCBAMClsHead(
                 in_channels=vit_backbone.embed_dim,
                 multiclass_loss=nn.CrossEntropyLoss(),
                 multilabel_loss=AsymmetricAngularLossWithIgnore(gamma_pos=0.0, gamma_neg=1.0, reduction="sum"),
+                step_size=1,
                 **head_config,
             ),
```

Comment:
Does this also work well without adding `optimize_gap` to `False`?

Reply:
Fix in [ed48f15](https://github.com/openvinotoolkit/training_extensions/pull/3810/commits/ed48f15864c74a8b14b3b5d7a68672d3e16f6ced)